stable/oauth2-proxy/templates/deployment.yaml

```diff
@@ -41,6 +41,13 @@ spec:
           - --{{ $key }}
           {{- end }}
         {{- end }}
+        {{- range $key, $value := .Values.dupExtraArgs }}
```

Comment:
This only allows for adding a second copy of ExtraArgs, which would fall apart if we need to add a 3rd or 4th validated email domain.

Possible good solutions:
- We could have ExtraArgs introspect the value type, and emit multiple copies when necessary
- We could pull whitelisted email domains out of extraArgs, and set it from a list value directly.